packages/common/src/items/setItemThumbnail.ts

```diff
@@ -13,12 +13,14 @@ export async function setItemThumbnail(
   id: string,
   file: any,
   filename: string,
-  requestOptions: IUserRequestOptions
+  requestOptions: IUserRequestOptions,
+  owner?: string
```

Comment:
Should `owner` really be optional? As it's implemented now, since we don't provide `owner `as a standalone `opt` or on the `item`, the `updateItem` call falls back to using the `authentication.username` which (as we saw) causes false failures.

I'd argue we should make this a required arg; however, if we do that, I understand this is technically a breaking change...although I don't believe it will have far-reaching consequences.